docs/README.md

```diff
@@ -13,6 +13,7 @@ To edit documentation:
 
   Don't forget you need to add your file to the `toctree` in `/docs/source/index.rst` *and* `/docs/configs/pdf/index.rst`.
 - **Make sure you preview** before you push.
+- Don't insert line-breaks inside inline literals. Building preview will treat this as an error.
```

Comment:
I'm not fully convinced this is a good idea. Now, you can't pipe your paragraphs through things like `fmt -80` anymore. But maybe we should not have line breaks in paragraphs (or at least sentences) anyway.

Reply:
It's a tradeoff between our convenience when writing docs, implementing a check, and having error-free docs. This is the middle ground imo. If you want to invest more, one could run a check on entire files validating that single, double, and triple backticks come in matching pairs. That's clearly more implementation effort. Or we can do without a check altogether, but given how many issues this uncovered, I'm in favour of having it.